.vscode/extensions.json

```diff
@@ -1,7 +1,5 @@
 {
     "recommendations": [
-        "dbaeumer.vscode-eslint",
```

Comment:
Removing those old plugin recommendations so that the official msft vscode eslint plugin get picked up.